### PR TITLE
Add falco to Kubernetes modules

### PR DESCRIPTION
**Pull request by phillebaba**

### CODE REVIEW

File: modules/kubernetes/falco/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+| kubernetes | 1.13.3 |
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| datadog\_api\_key | Datadog api key used to authenticate | `string` | n/a | yes |
+| datadog\_site | Datadog host to send events to | `string` | `"api.datadoghq.eu"` | no |
+| environment | Variable to add to custom fields | `string` | n/a | yes |
+| minimum\_priority | Minimum priority required before being exported | `string` | `"INFO"` | no |
```
Comment (simongottschlag) [inline modules/kubernetes/falco/README.md:29]: Should we have warning instead?

File: modules/kubernetes/falco/main.tf [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    minimum_priority = var.minimum_priority
+    datadog_host     = "https://${var.datadog_site}"
+    datadog_api_key  = var.datadog_api_key
+  })
+}
+
+resource "kubernetes_namespace" "this" {
+  metadata {
+    labels = {
+      name = "falco"
```
Comment (simongottschlag) [inline modules/kubernetes/falco/main.tf:37]: Add namespace to opa-gatekeeper exclusions?

**Review by simongottschlag — APPROVED** [review]
LGTM

File: modules/kubernetes/falco/main.tf (hunk 2 above)
Comment (phillebaba) [reply modules/kubernetes/falco/main.tf:37]: Added

**Pull request merged**